libraries/streaming-e2e-tests/react-app/src/App.js

```diff
@@ -3,7 +3,7 @@ import ReactWebChat from 'botframework-webchat';
 
 function App({directLine}) {
   return (
-    <ReactWebChat directLine={directLine} />
+    <ReactWebChat directLine={directLine} nonce="a1b2c3d" />
```

Comment:
Related to https://github.com/microsoft/botbuilder-js/pull/2763/files#r488188038

Reply:
Done: comment added